pyuploadcare/api_resources.py

```diff
@@ -277,7 +277,7 @@ def construct_from(cls, file_info):
         return file_
 
     @classmethod
-    def upload(cls, file_obj, store=None):
+    def upload(cls, file_obj, store=None, signature=None, expire=None):
```

Comment:
if we have `conf.signed_uploads` we don't have to change the method signature.
but we have to add an expire to conf or as an argument.